app/controllers/AddABeneficiaryController.scala

```diff
@@ -71,12 +68,21 @@ class AddABeneficiaryController @Inject()(
 
             val beneficiaryRows = new AddABeneficiaryViewHelper(all).rows
 
-            Ok(addAnotherView(
-              form = addAnotherForm,
-              inProgressBeneficiaries = beneficiaryRows.inProgress,
-              completeBeneficiaries = beneficiaryRows.complete,
-              heading = all.addToHeading
-            ))
+            if (beneficiaries.availableOptions.isEmpty) {
```

Comment:
Is there a better name for this property? Wasn't clear initially what was meant by availableOptions?